packages/ui-components/src/components/AppShell/AppShell.test.tsx

```diff
@@ -0,0 +1,175 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+// // TODO: When we the other components migrate to TS we will migrate this test in the end and delete this dummy
```

Comment:
Is it really necessary to comment out all the tests for this component? We have an issue planned for this sprint which will touch the AppShell. It's not great if all the tests are turned off.

Maybe it would have been easier to first migrate a component that's not used in the tests of a component that hasn't been migrated yet?

Reply:
Make sense. Could you suggest one that is easiest and not depends to others(not imported), that I can also easily check on a Example or Storybook page?